src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -52,36 +51,73 @@ public static ImmutableArray<BasicBlock> Create(IBlockOperation body)
             return blocks.ToImmutableAndFree();
         }
 
+        /// <summary>
+        /// Do a pass to eliminate blocks without statements that can be merged with predecessor(s).
+        /// </summary>
+        /// <param name="blocks"></param>
```

Comment:
Thanks. I feel this comment is clearer than old one.
`<param` line can probably be removed.